manifests/a/AxisCommunicationsAB/AXISCameraStation/5.52.56956/AxisCommunicationsAB.AXISCameraStation.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AxisCommunicationsAB.AXISCameraStation
+PackageVersion: 5.52.56956
+PackageLocale: en-US
+Publisher: Axis Communications AB
+PackageName: AXIS Camera Station
+License: Proprietary
+Copyright: Copyright © 2007-2023 Axis Communications AB
+ShortDescription: AXIS Camera Station

```

Comment:
Please provide a better description of what Axis Camera Station is used for

Reply:
AXIS Camera Station is a video management and access management software especially developed to fit a wide range of installations.